pkg/controller/acicontainersoperator.go

```diff
@@ -127,6 +143,51 @@ func NewAciContainersOperator(
 		0,
 		cache.Indexers{},
 	)
+	node_informer := cache.NewSharedIndexInformer(
```

Comment:
should we do all the node processing also conditionally?

Reply:
no not required really. if tomorrow something comes we can handle it

Reply:
Added additional check for flavor in the function triggered by watcher